set.mm

```diff
@@ -274682,20 +274674,18 @@ the same dimension over the same (nonzero) ring.  (Contributed by AV,
     sraassa.a $e |- A = ( ( subringAlg ` W ) ` S ) $.
     $( The subring algebra over a commutative ring is an associative algebra.
        (Contributed by Mario Carneiro, 6-Oct-2015.) $)
-    sraassa $p |- ( ( W e. CRing /\ S e. ( SubRing ` W ) ) ->
-      A e. AssAlg ) $=
+    sraassa $p |- ( ( W e. CRing /\ S e. ( SubRing ` W ) ) -> A e. AssAlg ) $=
```

Comment:
I suppose this will need to be revised later, too, to remove the commutativity requirement?

Reply:
I opened a new issue for this (#4693). I thought it would resolve this thread like in gitlab but it didn't, so I'm resolving it manually.